moveit_kinematics/kdl_kinematics_plugin/src/kdl_kinematics_plugin.cpp

```diff
@@ -228,7 +228,10 @@ bool KDLKinematicsPlugin::initialize(const std::string& robot_description, const
   for (std::size_t i = 0; i < kdl_chain_.getNrOfSegments(); ++i)
   {
     const robot_model::JointModel* jm = robot_model_->getJointModel(kdl_chain_.segments[i].getJoint().getName());
-
+    if (jm == nullptr)
+    {
```

Comment:
Should parentheses be omitted?

Reply:
The moveit code style follows the ROS code style. (http://wiki.ros.org/CppStyleGuide)
Here it says that if the block has only one line, you can omit the parentheses, but it does not say that it is required.
Therefore, it does not violate the code rules and standards, so we do not think it needs to be modified.